prov/efa/src/rxr/rxr_pkt_type_base.c

```diff
@@ -405,8 +408,12 @@ int rxr_pkt_copy_data_to_cuda(struct rxr_ep *ep,
 		if (rx_entry->cuda_copy_method == RXR_CUDA_COPY_GDRCOPY)
 			return rxr_pkt_queued_copy_data_to_hmem(ep, pkt_entry, data, data_size, data_offset);
 
-		/* If there are still empty slot for using gdrcopy, use gdrcopy on this rx_entry */
-		if (rx_entry->cuda_copy_method == RXR_CUDA_COPY_UNSPEC && ep->gdrcopy_rx_entry_num < max_gdrcopy_rx_entry_num) {
```

Comment:
I need to think about this change more.

Reply:
Actually if we're abandoning GDRCopy no non-p2p, this commit can be discarded altogether. Unless we want to keep it in to support later.

Reply:
In that case, let's remove this commit from this PR first. I will exam the code, and think about it more.